DXRMapFixups/DeusEx/Classes/DXRFixupParis.uc

```diff
@@ -229,6 +252,18 @@ function PreFirstEntryMapFixes()
                 }
 
             }
+        } else {
+            //Items in an unopenable shop window (Nutella store)
+            foreach RadiusActors(class'Actor',a,100,vect(-4635,-4110,280)){
```

Comment:
should be... just in case? also the one below

```suggestion
            foreach RadiusActors(class'Actor',a,100,vectm(-4635,-4110,280)){
```

Reply:
ah, yeah, guess so